### PR TITLE
nginx: adds ubus external module

### PR DESCRIPTION
**Pull request by Ansuel**
- Adds ubus module to nginx
- Select module by default
- Adds script to detect ubus module and append config for luci

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

File: net/nginx/Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nginx
 PKG_VERSION:=1.16.1
-PKG_RELEASE:=2
+PKG_RELEASE:=4
```
Comment (karlp) [inline net/nginx/Makefile]: you probably only meant 3 here?

**Comment by Ansuel** [comment]
There is another pending pr for nginx

Il Ven 25 Ott 2019, 12:36 Karl Palsson <notifications@github.com> ha
scritto:

**Comment by jow-** [comment]
Any reason why you're not following the `uhttpd-mod-ubus` protocol?

**Comment by Ansuel** [comment]
What do you mean?
If we have nginx we can't use uhttpd-mod-ubus as we don't have uhttpd
installed

Il Ven 25 Ott 2019, 12:44 Jo-Philipp Wich <notifications@github.com> ha
scritto:

**Comment by jow-** [comment]
> What do you mean?

I am talking about the wire protocol, the actual JSON that is sent and received by LuCI.

**Comment by Ansuel** [comment]
It's all based on uhttpd module so it does support every Luci request
In fact I test it on Luci

Il Ven 25 Ott 2019, 13:24 Jo-Philipp Wich <notifications@github.com> ha
scritto:

**Comment by Ansuel** [comment]
Don't know why build failed...

**Comment by neheb** [comment]
Needs a rebase.

**Comment by Ansuel** [comment]
@hnyman can this be merged?

**Pull request merged**